docs/core/compatibility/windows-forms/8.0/exceptioncollection.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking change: ExceptionCollection constructor throws ArgumentException"
+description: Learn about the breaking change in .NET 8 for Windows Forms where the ExceptionCollection constructor now throws an exception if the input is not of type Exception.
+ms.date: 02/06/2023
+---
+# ExceptionCollection ctor throws ArgumentException
+
+The <xref:System.ComponentModel.Design.ExceptionCollection> constructor now throws an <xref:System.ArgumentException> if the elements in the input array are not of type <xref:System.Exception>.
+
+## Version introduced
+
+.NET 8 Preview 1
+
+## Previous behavior
+
+Previously, the <xref:System.ComponentModel.Design.ExceptionCollection> constructor did not check the type passed in, which could delay failure until later in the process. No exceptions were thrown during object creation.
```

Comment:
Not sure, but since it's a collection, it may be different type_s_?

```suggestion
Previously, the <xref:System.ComponentModel.Design.ExceptionCollection> constructor did not check the types passed in, which could delay failure until later in the process. No exceptions were thrown during object creation.
```

Reply:
> Not sure, but since it's a collection, it may be different type_s_?

Input is `ArrayList` of type Exception.